sdk/documenttranslation/Azure.AI.DocumentTranslation/src/DocumentTranslationClient.cs

```diff
@@ -177,7 +177,7 @@ public virtual DocumentTranslationOperation StartTranslation(Uri sourceBlobConta
 
             var target = new TranslationTarget(targetBlobContainerSas, targetLanguageCode)
             {
-                Category = options?.Category
+                CategoryId = options?.Category
```

Comment:
Should we rename it to CategoryId in options too?

Reply:
Hmm good point. I didn't modify it there because we are looking at redesigning the input logic and `Options` will probably go away

Reply:
Yes I guess you're right we'll probably remove it anyway